elyra/metadata/metadata_app_utils.py

```diff
@@ -640,10 +640,10 @@ def print_subcommands(self):
         print("-----------")
         print("Subcommands are launched as `elyra-metadata cmd [args]`. For information on")
         print("using subcommand 'cmd', run: `elyra-metadata cmd -h` or `elyra-metadata cmd --help`.")
-        print("\nFind more information at https://elyra.readthedocs.io/en/latest/")
```

Comment:
Same for this file

Reply:
I did not explicitly change this file. Feel free to make any changes to it.